backend/src/process.py

```diff
@@ -628,6 +642,10 @@ async def update_progress():
                 self.cache_strategy[collector_node.id],
             )
 
+        if iterator_output.iterator.defer_errors and len(errors) > 0:
```

Comment:
Last thing: instead of checking `iterator_output.iterator.defer_errors` again here, could we rename `errors` to `deferred_errors` and omit the `iterator_output.iterator.defer_errors` check?

Reply:
Sure